testlib/helper.go

```diff
@@ -169,8 +170,7 @@ func (h *MainHelper) PreloadMigrations() {
 	handle := h.SQLStore.GetMaster()
 	_, err = handle.Exec(string(buf))
 	if err != nil {
-		mlog.Error("Error preloading migrations. Check if you have &multiStatements=true in your DSN if you are using MySQL. Or perhaps the schema changed? If yes, then update the warmup files accordingly.")
-		panic(err)
+		panic(errors.Wrap(err, "error preloading migrations. check if you have &multiStatements=true in your DSN if you are using MySQL. or perhaps the schema changed? if yes, then update the warmup files accordingly"))
```

Comment:
This is a bit tricky. I realize error strings should not be capitalized. But we have 4 sentences here. It looks really odd if the next letter is small after a period. Let's keep using capitals as an exception here.

Reply:
I was caught in the middle for this 😅